cypress/reporters/custom-spec-reporter.js

```
@@ -0,0 +1,349 @@
+        if (currentLine.length > 0) {
+            lines.push(currentLine);
+        }
+
+        return lines.length > 0 ? lines : [text];
+    }
+
+    function padRight(str, width) {
+        const strValue = String(str);
+        // Remove ANSI color codes to calculate the actual visible length
+        const visibleLength = strValue.replace(/\u001b\[\d+m/g, "").length;
+        const padding = width - visibleLength;
```

Biome (2.1.2)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)